executor/index_merge_reader.go

```diff
@@ -189,6 +189,7 @@ func (e *IndexMergeReaderExecutor) startPartialIndexWorker(ctx context.Context,
 		SetKeepOrder(false).
 		SetStreaming(e.partialStreamings[workID]).
 		SetFromSessionVars(e.ctx.GetSessionVars()).
+		SetMemTracker(e.memTracker).
```

Comment:
is this a bugfix?

Reply:
Probably yes. It infects the memory track for index merge reader in the index side.